noxfile.py

```diff
@@ -17,24 +17,30 @@
 
 from __future__ import absolute_import
 import os
-import shutil
-import sys
 import nox
 
 BLACK_PATHS = ["google", "tests"]
 
 if os.path.exists("samples"):
     BLACK_PATHS.append("samples")
 
+
 @nox.session
 def lint(session):
     """Run linters.
     Returns a failure if the linters find linting errors or sufficiently
     serious code quality issues.
     """
     session.install(
-        "flake8", "flake8-annotations", "black", "mypy", "sqlalchemy-stubs",
-        "types-pkg-resources", "types-PyMySQL", "twine")
+        "flake8==4.0.1",
+        "flake8-annotations==2.7.0",
+        "black==21.12b0",
+        "mypy==0.910",
+        "sqlalchemy-stubs==0.4",
+        "types-pkg-resources==0.1.3",
+        "types-PyMySQL==1.0.6",
+        "twine==3.7.1",
+    )
```

Comment:
Do you have a plan for how we want to update these? presumably renovate won't bump them. Should we put them in a different spot where it can?